Jenkinsfile

```diff
@@ -4,5 +4,16 @@ library("govuk")
 
 node {
   govuk.setEnvar("PUBLISHING_E2E_TESTS_COMMAND", "test-finder-frontend")
-  govuk.buildProject(sassLint: false, publishingE2ETests: true, brakeman: true, rubyLintDiff: false)
+  govuk.buildProject(
+    beforeTest: {
+      stage("Lint Javascript") {
+        sh("npm i standard")
+        sh("./node_modules/.bin/standard --global it --global expect --global beforeEach --global afterEach --global describe --global \$ --global GOVUK --global jasmine --global spyOn --global jQuery")
```

Comment:
Can you use eslint-env? we already have precedence for this in Content Publisher: https://github.com/alphagov/content-publisher/blob/f1587ab27002c01a45a17b7ded1b627b002b8954/spec/javascripts/components/contextual-guidance-spec.js#L1

It feels a bit fishy passing all those as globals across the entire codebase when most of those are jasmine too.

Reply:
Now done in 4478f5355f2c2e48d2c27035dd78b428c65bcc8d. Find it odd that your linked example uses both `eslint-env` and `global` in the same file - was there a reason for that?

I've used `global` as it reads better and seems to be less "vendor locked in" than the alternative, but happy to change if there's a good reason!

Reply:
eslint-env only knows about common environment globals: https://eslint.org/docs/user-guide/configuring#specifying-environments whereas GOVUK is one we've invented.

So the file in question is in both a jQuery and jasmine environment and then also uses a GOVUK global variable.

Best reason for eslint-env in the case of jasmine is that you don't need to keep adding to globals each time you use a different one of their functions.

Reply:
Thanks - makes sense. Have moved over to `eslint-env`.

Reply:
Awesome :+1: